### ISSUE TITLE
linter-flake8 not showing unused imports, unknown symbols and more

### ISSUE THREAD

**Issue by FalFire** [AtomLinter/linter-flake8#251]
I just installed linter-flake8 and linter-pep8 in Atom 1.10.2, and some errors show up but nothing about for instance unused imports and unknown symbols. When I manually run flake8 on my files it gives me a whole bunch of errors that do not show up in Atom. At first glance it looks like all messages starting with either D, F or H do not show up in Atom. Toggling the various settings do not have any effect, I currently left it at "Select Errors: E, W" and Flake Errors enabled.

**Comment by FalFire** [comment]
By coincidence I just figured it out. I first tried running `flake8 --version`and it outputted:

`2.5.5 (pep8: 1.5.7, pyflakes: 0.8.1, flake8-docstrings: 1.0.2, pydocstyle: 1.0.0, hacking.core: 0.0.1, ProxyChecker: 0.0.1, mccabe: 0.2.1) CPython 3.4.1 on Darwin`

I then ran `/usr/local/bin/flake8 --version` and it outputted:

`2.5.5 (pep8: 1.5.7, pyflakes: 0.8.1, flake8-docstrings: 1.0.2, pydocstyle: 1.0.0, hacking.core: 0.0.1, ProxyChecker: 0.0.1, mccabe: 0.2.1) CPython 2.7.10 on Darwin`

So the CPython versions are different and this apparently causes the described issue? When I set the path to flake8 in Atom the issues that were missing before immediately became visible.
